dotnetcore2.1/build/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM lambci/lambda-base:build
 
-ENV PATH=/var/lang/bin:$PATH \
+ENV DOTNET_ROOT=/var/lang/bin \
+    PATH=$DOTNET_ROOT:~/.dotnet/tools:$PATH \
```

Comment:
Now that it exists in an environment variable, I consolidated references to `/var/lang/bin`.